returnn/tf/layers/signal_processing.py

```diff
@@ -681,10 +737,15 @@ def __init__(self, nr_of_channels=1, **kwargs):
 
     input_placeholder = self.input_data.get_placeholder_as_batch_major()
 
-    output = tf.reshape(input_placeholder, [tf.shape(input_placeholder)[0], tf.shape(input_placeholder)[1], nr_of_channels, tf.shape(input_placeholder)[2] / nr_of_channels])
-    self.output.placeholder = tf.transpose(tf.reshape(tf.transpose(output, [1, 3, 0, 2]), (tf.shape(output)[1], tf.shape(output)[3], tf.shape(output)[0] * tf.shape(output)[2])), [2, 0, 1])
+    output = tf.reshape(input_placeholder,
+                        [tf.shape(input_placeholder)[0], tf.shape(input_placeholder)[1], nr_of_channels,
+                         tf.shape(input_placeholder)[2] / nr_of_channels])
+    self.output.placeholder = tf.transpose(tf.reshape(tf.transpose(output, [1, 3, 0, 2]), (
+      tf.shape(output)[1], tf.shape(output)[3], tf.shape(output)[0] * tf.shape(output)[2])), [2, 0, 1])
     # work around to obtain result like numpy.repeat(size_placeholder, nr_of_channels)
-    self.output.size_placeholder = {self.output.time_dim_axis_excluding_batch: tf.reshape(tf.tile(tf.reshape(self.input_data.size_placeholder[self.input_data.time_dim_axis_excluding_batch], [-1, 1]), [1, nr_of_channels]), [-1])}
+    self.output.size_placeholder = {self.output.time_dim_axis_excluding_batch: tf.reshape(
+      tf.tile(tf.reshape(self.input_data.size_placeholder[self.input_data.time_dim_axis_excluding_batch], [-1, 1]),
+              [1, nr_of_channels]), [-1])}
```

Comment:
```suggestion
    self.output.size_placeholder = {
      self.output.time_dim_axis_excluding_batch: ...}
```
This is also way too complicated. Isn't this just the same as `tile_transposed`?

Reply:
I split this into several lines to make it more readable.

Reply:
Ok for now. But I think further work on this should simplify this. If this is just `tile_transposed`, you should just use this function.